media/media.js

```diff
@@ -58,27 +68,27 @@ document.querySelectorAll('div.card.project').forEach((box) =>
     switch (states.repoType) {
       case 'piral':
         document.querySelectorAll('div.onlyForPilet').forEach((box) => {
-          box.classList.remove('hide');
-          box.classList.add('hide');
+          display(box);
+          hide(box);
           states.piralPackage = '';
           states.npmRepository = '';
         });
         document.querySelectorAll('div.piral-template').forEach((box) => {
```

Comment:
This is all unnecessary verbose. If you do `(box) => { display(box); }` you can also do `(box) => display(box)`, which means you can do `display`.

So just use

```js
document.querySelectorAll('div.piral-template').forEach(display);
```

(this is the same for all things here; you can do the same with `hide`.)